learningresources/tests/base.py

```diff
@@ -35,6 +35,13 @@ def logout(self):
         """just a client logout"""
         self.client.logout()
 
+    def assert_status_code(self, url, code, return_body=False):
+        """Asserts the status code of GET attempt"""
+        resp = self.client.get(url, follow=True)
+        self.assertTrue(resp.status_code == code)
+        if return_body:
```

Comment:
This isn't necessary; it's okay to always return the body and let the caller decide whether to ignore it. That doesn't break pep8 or pylint, right?